application/gqlgen/user.go

```diff
@@ -234,3 +234,16 @@ func (r *userResolver) Preferences(ctx context.Context, obj *models.User) (*mode
 
 	return &standardPrefs, nil
 }
+
+// getPublicProfile converts a models.User to a PublicProfile, which hides private profile information
+func getPublicProfile(user *models.User) (*PublicProfile, error) {
+	url, err := user.GetPhotoURL()
+	if err != nil {
+		return nil, err
```

Comment:
Just wondering if it would be better to only log the error and  not return an error here.